### PR TITLE
chore: master merge feature

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [x] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### CODE REVIEW

**Comment by li-jia-nan** [comment]
argos 有点多……

**Comment by afc163** [comment]
又大了 1kB ..

**Comment by zombieJ** [comment]
ComponentToken 不能压，出入都会消耗 bundle size

File: package.json [hunk 1]
```diff
@@ -113,20 +113,21 @@
     "@ant-design/react-slick": "~1.0.0",
     "@babel/runtime": "^7.18.3",
     "@ctrl/tinycolor": "^3.6.0",
+    "@rc-component/color-picker": "^1.0.0-alpha.11",
```
Comment (yoyo837) [inline package.json:116]: 这个没升级

Comment (yoyo837) [reply package.json:116]:
```suggestion
    "@rc-component/color-picker": "~1.0.0",
```

**Pull request merged**